doc/man1/openssl-x509.pod.in

```diff
@@ -396,21 +398,20 @@ generate a certificate containing any desired public key.
 =item B<-clrext>
 
 When a transforming a certificate to a new certificate
```

Comment:
When you're editing here please s/a transforming/transforming/

Reply:
Done.